### PR TITLE
refactor(api): Port `ReadAbsorbanceImpl` and `FileStore` to `StateUpdate`

### PR DESCRIPTION
**Pull request by SyntaxColoring**
# Overview

Closes https://opentrons.atlassian.net/issues/EXEC-787.

## Test Plan and Hands on Testing

Neither `ReadAbsorbanceImpl` nor `FileStore` have unit tests. I think this is a trivial enough refactor that we can get by with only code review, but I can add tests if anyone's uncomfortable with that.

## Changelog

Port `ReadAbsorbanceImpl` and `FileStore` to the new `StateUpdate` mechanism.

## Review requests

None.

## Risk assessment

Low.

### CODE REVIEW

File: api/tests/opentrons/protocol_engine/commands/test_move_to_well.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ async def test_move_to_well_stall_defined_error(
     error_id = "error-id"
     error_timestamp = datetime(year=2020, month=1, day=2)
     decoy.when(
-        movement.move_to_well(
+        await movement.move_to_well(
```
Comment (SyntaxColoring) [inline api/tests/opentrons/protocol_engine/commands/test_move_to_well.py:129]: Unrelated change, just fixing a missing `await`.

**Review by TamarZanzouri — APPROVED** [review]
LGTM!
